### PR TITLE
Servicebus: updated for GEO DR

### PR DESCRIPTION
**Pull request by v-Ajnava**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json [hunk 1]
```diff
@@ -692,6 +692,245 @@
...
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```
Comment (ravbhatnagar) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: Please use 202 for delete LRO.

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json [hunk 2]
```diff
@@ -3421,6 +3670,68 @@
         }
       },
       "description": "Capture storage details for capture description"
+    },
+    "ArmDisasterRecovery": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "properties": {
+            "provisioningState": {
```
Comment (ravbhatnagar) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: make this enum

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json [hunk 3]
```diff
@@ -692,6 +692,245 @@
...
+          "default": {
+            "description": "ServiceBus error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/disasterRecoveryConfig/{alias}/breakpairing": {
```
Comment (ravbhatnagar) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: breakPairing (camelcase)

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json [hunk 4]
```diff
@@ -692,6 +692,245 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/disasterRecoveryConfig": {
```
Comment (ravbhatnagar) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: pluralize resource type name? disasterRecoveryConfigs.

**Review by ravbhatnagar — COMMENTED** [review]
@v-Ajnava - Some feedback for you

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json (hunk 2 above)
Comment (v-Ajnava) [reply specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: Resolved, we don't support cancel.

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json (hunk 3 above)
Comment (v-Ajnava) [reply specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: Resolved

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json (hunk 1 above)
Comment (v-Ajnava) [reply specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: Resolved

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json (hunk 4 above)
Comment (v-Ajnava) [reply specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: Resolved

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json (hunk 1 above)
Comment (v-Ajnava) [reply specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json]: Reverted as delete is not async

**Comment by salameer** [comment]
@sarangan12  can you add your feedback here

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**